src/components/layout/footer/main-links.js

```diff
@@ -87,10 +88,10 @@ const MainLinksSection = ({ is_ppc, is_ppc_redirect }) => {
                         <LinkWrapper>
                             <Link to="/markets/forex/">{<Localize translate_text="Forex" />}</Link>
                         </LinkWrapper>
-                        {is_non_uk && !is_ppc && (
+                        {is_non_uk && (
                             <LinkWrapper>
-                                <Link to="/markets/synthetic/">
-                                    {<Localize translate_text="Synthetic indices" />}
+                                <Link to="/markets/derived-fx/">
```

Comment:
since we change the URL from `.../synthetic/` to `.../derived-fx`, shouldn't we add `nginx` redirection from old URL to the new one?

Reply:
I don't think so since synthetic indices is not being removed but rather being a submarket of Derived.

Reply:
ok cool, thanks :pray: